juju/juju
# Lock gometalinter as a dependency

## Description of change

> Why is this change needed?

The following commit locks the gometalinter as a dependency, so any
upstream changes to the gometalinter don't potentially break the
juju release process. In order to do this, we have to force the
gometalinter into the juju repository. We do this by making a test
that checks what the output of the bash file is looking for. This
should protect us when upgrading gometalinter in the future, by
erroring out if there are differences.

## QA steps

> How do we verify that the change works?

Check locally and with the CI server bots that everything is green

## Documentation changes

> Does it affect current user workflow? CLI? API?

No, it should be transparent.

## Bug reference

> Does this change fix a bug? Please add a link to it.

No, this was a change recommended by @howbazaar
